e2e/tests/hmctsCompliesOnBehalfOfOtherPartiesAfterSDOIsIssued_test.js

```diff
@@ -6,56 +6,42 @@ let caseId;
+  I.seeEventSubmissionConfirmation(config.administrationActions.sendToGatekeeper);
 
-    //hmcts login, add case number, add hearing details, allocated judge and send to gatekeeper
-    await I.signIn(config.hmctsAdminEmail, config.hmctsAdminPassword);
-    await I.navigateToCaseDetails(caseId);
-    await caseViewPage.goToNewActions(config.administrationActions.addFamilyManCaseNumber);
-    enterFamilyManCaseNumberEventPage.enterCaseID();
-    await I.completeEvent('Save and continue');
-    await caseViewPage.goToNewActions(config.administrationActions.addHearingBookingDetails);
-    await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[0]);
-    await I.completeEvent('Save and continue', {summary: 'summary', description: 'description'});
-    await caseViewPage.goToNewActions(config.administrationActions.sendToGatekeeper);
-    sendCaseToGatekeeperEventPage.enterEmail();
-    await I.completeEvent('Save and continue');
-    I.seeEventSubmissionConfirmation(config.administrationActions.sendToGatekeeper);
-    I.signOut();
+  await I.navigateToCaseDetailsAs(config.gateKeeperUser, caseId);
 
-    //gatekeeper login, draft sdo and select issued
-    await I.signIn(config.gateKeeperEmail, config.gateKeeperPassword);
-    await I.navigateToCaseDetails(caseId);
-    await caseViewPage.goToNewActions(config.applicationActions.allocatedJudge);
-    await allocatedJudgeEventPage.enterAllocatedJudge('Moley');
-    await I.completeEvent('Save and continue');
-    await caseViewPage.goToNewActions(config.administrationActions.draftStandardDirections);
-    await draftStandardDirectionsEventPage.skipDateOfIssue();
-    await draftStandardDirectionsEventPage.useAllocatedJudge('Bob Ross');
-    await draftStandardDirectionsEventPage.enterDatesForDirections(directions[0]);
-    draftStandardDirectionsEventPage.markAsFinal();
-    await I.completeEvent('Save and continue');
-    I.seeEventSubmissionConfirmation(config.administrationActions.draftStandardDirections);
-    I.signOut();
-  }
+  await caseViewPage.goToNewActions(config.applicationActions.allocatedJudge);
+  await allocatedJudgeEventPage.enterAllocatedJudge('Moley');
+  await I.completeEvent('Save and continue');
+  await caseViewPage.goToNewActions(config.administrationActions.draftStandardDirections);
+  await draftStandardDirectionsEventPage.skipDateOfIssue();
+  await draftStandardDirectionsEventPage.useAllocatedJudge('Bob Ross');
+  await draftStandardDirectionsEventPage.enterDatesForDirections(directions[0]);
+  draftStandardDirectionsEventPage.markAsFinal();
+  await I.completeEvent('Save and continue');
 });
 
-Scenario('hmcts admin complies with directions on behalf of other parties', async (I, caseViewPage, complyOnBehalfOfOthersEventPage) => {
-  await I.signIn(config.hmctsAdminEmail, config.hmctsAdminPassword);
-  await I.navigateToCaseDetails(caseId);
+Before(async I => await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId));
```

Comment:
Do we need to do it (changing user) before every test? I am afraid if smoeone adds new test this might be not required.

Reply:
agree, done